src/platform/nrfconnect/wifi/WiFiManager.h

```diff
@@ -173,6 +173,7 @@ class WiFiManager
     static constexpr uint32_t kConnectionRecoveryMaxIntervalMs     = CONFIG_CHIP_WIFI_CONNECTION_RECOVERY_MAXIMUM_INTERVAL;
     static constexpr uint32_t kConnectionRecoveryJitterMs          = CONFIG_CHIP_WIFI_CONNECTION_RECOVERY_JITTER;
     static constexpr uint32_t kConnectionRecoveryMaxRetries        = CONFIG_CHIP_WIFI_CONNECTION_RECOVERY_MAX_RETRIES_NUMBER;
+    static constexpr uint32_t kSupplicantReconnectionTimeoutMs     = 60000;
```

Comment:
It is defined somewhere, measured, or just set regarding observations?

Reply:
It's just something we need to decide. I don't think there are any measurements we could do. This is the value that defines how long time are we ok to give supplicant to fix our connection. After this time we will disconnect and start re-scanning based on our recovery mechanism.

Reply:
If you or @markaj-nordic have some opinion regarding this value we can change it. 60 s in my opinion was big enough to give module a time to reconnect during usual network hiccups and not too big, so user would be annoyed that the device is not available for too long in case supplicant would stuck

Reply:
Yehh, I don't have a strong opinion on that, maybe from the user perspective 60s is better then 120 :)